server/game/cards/10-TEL/MotoAriq.js

```diff
@@ -0,0 +1,27 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+const { Players, CardTypes } = require('../../Constants');
+
+class MotoAriq extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Move a ready character to the conflict',
+            condition: context => context.game.isDuringConflict()
```

Comment:
You don't need `context.game.isDuringConflict()` here

Reply:
Makes sense, since participating and the moveToConflict action already require a conflict. Fixed